ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -81166,7 +81166,7 @@ periodics:
     ci.openshift.io/no-builds: "true"
     job-release: "4.15"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-release-master-nightly-4.15-e2e-agent-single-node-ipv6
+  name: periodic-ci-openshift-release-master-nightly-4.15-e2e-agent-single-node-ipv6-conformance
```

Comment:
The reason why the `release-controller-config` test failed is because you've changed the name of a job that's being used for release verification.  You'll need to add the rename to these files as well:
https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.15.json#L47-L52
https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.16.json#L47-L52
https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.17.json#L39-L44

Reply:
Thank you @bradmwilliams